app/views/users/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>All Users</h1>
\ No newline at end of file
+<section class="users-container">
+<ul>
+<% @users.each do |user|%>
+<li class="user-li">
+<a href="<%= user_path(:id => user.id) %>">
+  <div class="user-card">
+    <div class="image-container">
+    <img src = <%= user.photo %>>
+    </div>
+    <h2><%= user.name %></h2>
+  </div>
+  <p class="post-count">Number of Posts:<%= user.posts_counter%></p>
+  </a>
+</li>
+<% end %>
+
+</ul>
+
+</section>
```

Comment:
- Hey, after creating a user, the user doesn't seem to be created and it's empty in the console. please, kindly check this to fix this issue.
- And this has affected every other view, e.g, show, post index, and post-show page. kindly fix it to ensure each page shows as expected.

**NOTE** _Please, kindly fixes all the pages and ensure they work as expected!_

Reply:
thank you @zieeco for suggestion I will work on fixing the bug.